docs/README.md

```diff
@@ -1,11 +0,0 @@
-# Getting Started with Docs
```

Comment:
As long as there is some equivalent pointer in the docs directory removing this is okey doke. The GSoC students didn't have a clear place to "start", and I put this in here because they were looking for guidance in this directory.

Reply:
As there has been since the beginning of the summer, there's a whole page about how to write and build our documentation. We should not be using markdown in the `docs` folder, especially if the content already exists.